README.md

```diff
@@ -298,6 +298,7 @@ Table of Contents
   * [Crystallize](https://crystallize.com) — Headless PIM with ecommerce support. Built-in GraphQL API. Free version includes unlimited users, 1000 catalogue items, 5 GB/month bandwidth and 25k/month API calls.
   * [Forestry.io/](https://forestry.io/) — Headless CMS. Give your editors the power of Git. Create and edit Markdown-based content with ease. Comes with three free sites that includes 3 editors, Instant Previews. Integrates with blogs hosted on Netlify/GitHubpages/ elsewhere
   * [kontent.ai](https://www.kontent.ai) - A Content-as-a-Service platform that gives you all the headless CMS benefits while empowering marketers at the same time. Developer plan provides 2 users with unlimited projects with 2 environments for each, 500 content items, 2 languages with Delivery and Management API, and Custom elements support. Larger plans available to meet your needs.
+  * [NetlifyCMS](https://www.netlifycms.org/) - Headless CMS, git-based. You can add it to a site that's built with a common static site generator, like Jekyll, Hugo, NextJS, Gatsby... Alternatively, there are [several starter templates](https://www.netlifycms.org/docs/start-with-a-template/) that you can set-up and deploy with Netlify.
```

Comment:
NetlifyCMS is not a SaaS, NetlifyCMS is something that generates data to put on the SaaS (netlify). Feel free to mention the CMS in the existing Netlify entry, but as a CMS alone it would not quality for this list.

Reply:
I proposed it because because you can use NetlifyCMS without Netlify, as explained here: https://www.netlifycms.org/docs/intro/#netlify-cms-vs-netlify

Reply:
That makes it self hosted software, not a SaaS, this list is for SaaS.

Reply:
Oh okay, I did not see this common point between the others.

Reply:
No problem at all :) Appreciate your contribution